sessionize/lib/src/iosMain/kotlin/co/touchlab/sessionize/platform/DateHelpers.kt

```diff
@@ -13,8 +17,29 @@ actual class Date(val iosDate: NSDate) {
 
 actual class DateFormatHelper actual constructor(format: String) {
 
-    private val formatter: NSDateFormatter = NSDateFormatter().apply { dateFormat = format }
+    private val dateFormatterConference: NSDateFormatter = NSDateFormatter().apply {
 
-    actual fun toDate(s: String): Date = Date(formatter.dateFromString(s)!!)
-    actual fun format(d: Date): String = formatter.stringFromDate(d.iosDate)
-}
\ No newline at end of file
+        val t = ServiceRegistry.timeZone
```

Comment:
No easy way to set time zone in iOS, so breaking down string variable into seconds.